sdk/communication/communication-network-traversal/src/communicationRelayClient.ts

```diff
@@ -156,11 +195,13 @@ export class CommunicationRelayClient {
    *
    * @param user - The user for whom to issue a token
    * @param routeType - The specified routeType for the relay request
+   * @param ttl - The specified Time to live for the relay credential
    * @param options - Additional options for the request.
    */
   public async getRelayConfiguration(
-    paramOne?: CommunicationUserIdentifier | RouteType | GetRelayConfigurationOptions,
-    paramTwo?: RouteType | GetRelayConfigurationOptions,
+    paramOne?: CommunicationUserIdentifier | RouteType | GetRelayConfigurationOptions | number,
```

Comment:
all this complicated logic isn't needed when we have only one function with one options bag